test/e2e/specs/test.js

```diff
@@ -7,6 +7,29 @@ module.exports = {
     .url('http://localhost:8080')
       .waitForElementVisible('#app', 5000)
       .assert.elementPresent('.randPersonBtn')
+      .pause(5000)
       .end()
+  },
+
+  'startship e2e tests': function (browser) {
+    browser
+        .url('http://localhost:8080')
+        .waitForElementVisible('#app', 5000)
+        //.click('.mdl-tabs__tab')
+        .setValue('.mdl-tabs__tab','Starship')
```

Comment:
Will this break if the layout or classes change? Would you recommend a different way?